### PR TITLE
added k-largest elements in array

### LINKED ISSUE #586: "K" largest element in an Array
#### Problem Statment:
Given an array Arr of N positive integers, find K largest elements from the array.  The output elements should be printed in decreasing order.

This can be solved using a Min-Heap.
I will be working on this 
I am a Gssoc'22 participant.

### Domain

Data Structure

### Type of Contribution

Addition

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) & [Code of conduct](https://github.com/Lakhankumawat/LearnCPP/blob/main/.github/CODE_OF_CONDUCT.md) of LearnCPP.

### PR DESCRIPTION
**Pull request by SidharthSarangi**
## Related Issue
This fixes #586 

### Problem Statement:
Given an array Arr of N positive integers, find K largest elements from the array.  The output elements should be printed in decreasing order.
[Link](https://www.geeksforgeeks.org/k-largestor-smallest-elements-in-an-array/)
### ADDITIONAL INFO
There was no Readme file in the repo/folder so I added the working of both max/min Heap along with time complexity and therefore updated the Readme.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
|Functional code with output vs expected output |
Screenshot (218)
|Test Cases Passed|
Screenshot (219)

### CODE REVIEW

**Comment by SidharthSarangi** [comment]
Please review @Lakhankumawat @siddhi-244 @iamgunjan

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]
You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Comment by Lakhankumawat** [comment]
Your Code looks okyish but modify your README

**Comment by SidharthSarangi** [comment]
check now, I have modified the Readme following the guidelines, and I am putting my 100% to make sure this repo gets completed with the proper resources. @Lakhankumawat

File: H-Heap/README.md [hunk 1]
```diff
@@ -1,19 +1,73 @@
...
-Get Max or Min Element
+**Insert an Element**  
+Time Complexity of this operation is O(Log n) because we insert the value at the end of the tree and traverse up to remove violated property of min/max heap.  
 
-The Time Complexity of this operation is O(1).
-Remove Max or Min Element
 
-The time complexity of this operation is O(Log n) because we need to maintain the max/mix at their root node, which takes Log n operations.
-Insert an Element
+# K LARGEST ELEMENTS
```
Comment (Lakhankumawat) [inline H-Heap/README.md:29]: follow camel casing , `K Largest Elements`

File: H-Heap/README.md [hunk 2]
```diff
@@ -1,19 +1,73 @@
-#### How are Min/Max Heaps represented?
+# TABLE OF CONTENTS
```
Comment (Lakhankumawat) [inline H-Heap/README.md:1]: Follow Camel Casing `Table of Contents`

File: H-Heap/README.md [hunk 3]
```diff
@@ -1,19 +1,73 @@
...
+* [Explanation](#explanation)  
+* [Time Complexity](#time-complexity-of-min-max-heap)  
+
+-[k-largest Elements in an Array](#k-largest-elements)
+## EXPLANATION
```
Comment (Lakhankumawat) [inline H-Heap/README.md:6]: Camel casing

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

File: H-Heap/README.md (hunk 1 above)
Comment (SidharthSarangi) [reply H-Heap/README.md]: done

File: H-Heap/README.md (hunk 2 above)
Comment (SidharthSarangi) [reply H-Heap/README.md]: done

File: H-Heap/README.md (hunk 3 above)
Comment (SidharthSarangi) [reply H-Heap/README.md]: done

**Comment by miraj0507** [comment]
This looks good to me.

**Review by Lakhankumawat — APPROVED** [review]
LGTM 💯👏

**Comment by Lakhankumawat** [comment]
@all-contributors please add @SidharthSarangi code,

**Pull request merged**